packages/backend/src/routes/courses.ts

```diff
@@ -25,7 +25,7 @@ coursesRouter.get("/:id", async (ctx, next) => {
     const { id } = ctx.params;
 
     try {
-        const rows = await new Promise((resolve, reject) => {
+        const course = await new Promise((resolve, reject) => {
```

Comment:
i think this should be courses since the query returns an array of rows? At least that's what the syntax suggests. Perhaps the default [] in db.get is wrong